trulens_eval/trulens_eval/feedback/provider/bedrock.py

```diff
@@ -63,17 +61,64 @@ def _create_chat_completion(
         assert self.endpoint is not None
         assert prompt is not None, "Bedrock can only operate on `prompt`, not `messages`."
 
-        # NOTE(joshr): only tested with sso auth
         import json
 
-        body = json.dumps({"inputText": prompt})
+        body = json.dumps({"inputText": prompt, "textGenerationConfig": {
+                "maxTokenCount": 4096,
```

Comment:
Make a task to make these parameters configurable in a future PR?

Reply:
@joshreini1 approved but see this note.

Reply:
Added a TODO for this